src/NovaClient.ts

```diff
@@ -30,7 +32,12 @@ export type NovaClientConfig = {
    * Password for basic auth to the Nova instance.
```

Comment:
We should indicate that this option is deprecated https://jsdoc.app/tags-deprecated